giraffe/package.json

```diff
@@ -9,8 +9,8 @@
     "build": "rm -rf dist && NODE_ENV=production rollup -c",
     "start": "rollup -cw",
     "lint": "eslint '{src,../stories/src}/**/*.{ts,tsx}'",
-    "prettier": "prettier --config ../.prettierrc.json --check '{.,../stories}/**/*.{ts,tsx}'",
-    "prettier:fix": "prettier --config ../.prettierrc.json --write '{.,../stories}/**/*.{ts,tsx}'",
+    "prettier": "prettier --config ../.prettierrc.json --check '{src,../stories/src}/**/*.{ts,tsx}'",
```

Comment:
generally i think it's probably better to pull these kinds of tweaks out of PRs and get them reviewed ahead of the logic changes in that PR so that we can separate out housekeeping pieces like this and logical pieces like the `key` addition.

i've noticed that when these types of changes get merged together, and there's a problem in the logic, and the whole thing gets reverted, the housekeeping bits tend to not get re-added. i think  people tend to think of the housekeeping parts as done already and forget they got reverted. i've done this a number of times and seen teammates do it as well.

not going to block on this, but something to consider doing, going forward.